en/products/relay/bundle.ftl

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/relay/
+
+# Variables:
+#   $monthly_price (string) - the monthly cost (including currency symbol) for a given plan. Examples: $0.99, 0,99 €
+bundle-banner-header-2 = Limited-time only: { -brand-name-relay-premium } + { -brand-name-mozilla-vpn } for { $monthly_price }/month
```

Comment:
Why the `-2` in the header? Since we're starting from scratch, we should get rid of versioning in the IDs.

Reply:
I won't be able to migrate string translations if we change IDs - even though its not ideal I would like to keep IDs the same where possible.